overlord/devicestate/handlers_install.go

```diff
@@ -92,3 +114,30 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 
 	return nil
 }
+
+// TODO:UC20: set to real TPM availability check function
+var checkTPMAvailability = func() error {
+	return nil
+}
+
+func checkEncryption(model *asserts.Model) (res bool, err error) {
+	secured := model.Grade() == asserts.ModelSecured
+
+	// check if we should disable encryption non-secured devices
+	if osutil.FileExists(filepath.Join(dirs.RunMnt, "ubuntu-seed", ".force-unencrypted")) {
```

Comment:
we really don't want even for just "signed" a non-signed mechanism to turn off encryption

Reply:
Changed to prevent disabling encryption if grade is secured or signed.